db/migrate/20221129162425_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] This table should refer to `:author`, this will create author_id and a corresponding index and then the foreign_key to the users' table. Please kindly fix that 🙃 😉